README.md

```diff
@@ -24,6 +35,10 @@
 
 It provides various tools and libraries to streamline the integration and interaction with decentralized protocols.
 
+## Path Releases
+
+Path releases provide a Docker image you can start using right away to bootstrap your Path gateway without the need of building your own image. Head over to [releases](https://github.com/buildwithgrove/path/releases) to download it.
```

Comment:
Can you provide an example command on how to download the docker image?

#PUC (Please Update Comment)

Reply:
This section has been updated.